pilot/lib/pilot/tester_manager.rb

```diff
@@ -74,7 +78,7 @@ def remove_tester(options)
           begin
             app = Spaceship::Application.find(app_filter)
             UI.user_error!("Couldn't find app with '#{app_filter}'") unless app
-            tester.remove_from_app!(app.apple_id)
+            app.default_external_group.remove_tester!(tester)
             UI.success("Successfully removed tester #{tester.email} from app #{app_filter}")
           rescue => ex
             UI.error("Could not remove #{tester.email} from app: #{ex}")
```

Comment:
We haven't fixed things as far as testers are concerned I don't think, so this might be better in a new PR that fixes tester management?